### ISSUE TITLE
Grapterodon Sound Ambient

### ISSUE THREAD

**Issue by Andresv2017** [ginerJ/Tameable-Beasts#4]
![imagen_2024-11-06_005415220](https://github.com/user-attachments/assets/fe54b016-f219-4d96-bcbc-b2638cd8c1fd)

![image](https://github.com/user-attachments/assets/dec73f7d-f3e6-4451-bc7d-556753195305)

Hey Bro, I noticed you forgot to use the Ambient Sound for the Grapterodon.  Your mod is incredible; it's very well-organized and clear. I'm reviewing it to see if anything else is missing. I wanted to ask for your permission to modify it so that when creatures breed without being tamed, they drop an egg instead of an entity. I wouldn't want to modify your code without your permission. Thanks.

**Comment by ginerJ** [comment]
Oh thanks so much! Will be sure to fix it on 6.2, gonna add it along a new model for the graptera which I'm not that happy about rn.

Feel free to edit as much as u want! And to suggest changes in case I do something dumb

**Comment by ginerJ** [comment]
No worries, got an idea an even a model for a marine creature (thought we could count the penguins as the marine mob of the mod if we are picky xd). Gonna start the graptera model today.

**Comment by Andresv2017** [comment]
I’ve been making progress on the water navigation system. It was a bit tricky to get everything working from RidingAnimal, but I managed to figure it out after a few days. I’m sending you a video with a demonstration.

https://drive.google.com/file/d/1_iObY5PZ2ajVMv2LEQalcL126Rq9dTl0/view?usp=sharing

I’ve set it so that pressing the 'X' key lets you descend in the water, and when mounted, the rider is positioned close to the head.

The model is a turtle that I simply enlarged, and the animations are basic. Honestly, I’m not very skilled at animating and managed the animations, but I think it does the job for now. If you have any suggestions on how to improve the animations, I’m all ears.

I’m open to any feedback or ideas you have on the changes I’ve made or anything else you’d like to see improved. Let me know what you think!"
